zerver/management/commands/purge_queue.py

```diff
@@ -5,19 +5,29 @@
 from django.core.management.base import BaseCommand
 from django.core.management import CommandError
 from zerver.lib.queue import SimpleQueueClient
+from zerver.worker.queue_processors import get_active_worker_queues
 
 class Command(BaseCommand):
     def add_arguments(self, parser):
         # type: (ArgumentParser) -> None
         parser.add_argument('queue_name', metavar='<queue name>', type=str,
                             help="queue to purge")
```

Comment:
I tried using this; you need to make this argument not required.

Reply:
Done that.